src/agent/border_agent.hpp

```diff
@@ -72,10 +72,23 @@ class BorderAgent
 
     ~BorderAgent(void);
 
+    /**
+     * This method updates the file descriptor sets with file descriptors used by border agent.
+     *
+     * @param[inout]  aReadFdSet   A reference to the read file descriptors.
+     * @param[inout]  aWriteFdSet  A reference to the write file descriptors.
```

Comment:
Thanks. Fixed this and other missing ones. Also changed a little `Doxyfile.in` to fix some unnecessary warnings.